ci/build/build-docker-images.py

```diff
@@ -97,16 +96,11 @@ def _get_branch():
 
 def _release_build():
     branch = _get_branch()
-    if branch is None:
-        return False
-    return branch != "master" and branch.startswith("releases")
```

Comment:
(note that, if it starts with "release", it certainly is not "master")

Reply:
😅 Yep :)